syntax = "proto3";

package PbData;

message SteerCommand
{
    double angleDegrees = 1;
}
